test/cases/orchestration.py

```diff
@@ -726,8 +726,6 @@ def reload_nginx(self, wait_for_workers_to_terminate=True):
                 "--",
                 "nginx",
                 "nginx",
-                "-g",
-                "pid /run/nginx.pid;",
```

Comment:
[question] May I ask why?

Reply:
When we build a openresty container, there is no /run/nginx.pid file, therefore all the tests are failing. Furthemore, i found out that removing this lines does not change anything to nginx tests. Therefore i just removed them

Reply:
This is mandatory for ingress-nginx which still passes for some reason on this branch. I noticed some errors in the test.

Reply:
Ok. I'll try to find another way